shopinvader_customer_validate/models/res_partner.py

```diff
@@ -85,6 +91,19 @@ def _display_validate_address(self):
         has_parent = self._get_has_shopinvader_parent()
         return has_parent and self.address_type == "address"
 
+    def _compute_backends_display(self):
+        for rec in self:
+            rec.backends_display = ", ".join(rec._get_valid_backends().mapped("name"))
+
+    def _get_valid_backends(self):
+        """Get valid backends from partner's hierarchy."""
+        partners = self
+        backends = partners.shopinvader_bind_ids.backend_id
+        while partners.parent_id:
+            backends |= partners.parent_id.shopinvader_bind_ids.backend_id
```

Comment:
Ss seen here, union isn't as efficient as browsing a set.
https://gist.github.com/guewen/a4a2eba5d128b4ed3129866f80571fbd

Reply:
I know :wink:

Reply:
That's probably why the build is :apple: 
:see_no_evil: